### ISSUE TITLE
Provide an APK to test quickly

### ISSUE THREAD

**Issue by ellis2323** [KamesCG/ethid-mobile#2]
Hello,

Is it possible to provide an APK to test quickly the product?

Best regards,

**Comment by KamesCG** [comment]
Yes. I will get the APKs and App Store prototypes ready in the next 1-2 weeks.
